Makefile

```diff
@@ -1,3 +1,18 @@
+# Current Operator version
+VERSION_DATE ?= $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
+VERSION_PKG ?= "github.com/os-observability/tempo-operator/internal/version"
+TEMPO_VERSION ?= "1.5.0"
+OPERATOR_VERSION ?= 0.0.1
+COMMIT_SHA = "$(shell git rev-parse HEAD)"
+LD_FLAGS ?= "-X ${VERSION_PKG}.buildDate=${VERSION_DATE} -X ${VERSION_PKG}.version=${OPERATOR_VERSION} -X ${VERSION_PKG}.commitSha=${COMMIT_SHA}"
+ARCH ?= $(shell go env GOARCH)
+
+# Image URL to use all building/pushing image targets
+IMG_PREFIX ?= ghcr.io/${USER}/tempo-operator
+IMG_REPO ?= tempo-operator
+IMG ?= ${IMG_PREFIX}/${IMG_REPO}:${OPERATOR_VERSION}
+BUNDLE_IMG ?= ${IMG_PREFIX}/${IMG_REPO}-bundle:${OPERATOR_VERSION}
```

Comment:
Should we prefix docker (version) tags with `v`? Currently `CATALOG_IMG` has the prefix, `BUNDLE_IMG` and `IMG` doesn't. imho either all of them should have the `v` prefix, or none.